packages/tabs/src/styled/StyledTab.ts

```diff
@@ -8,31 +8,36 @@
 import styled, { DefaultTheme, css, ThemeProps } from 'styled-components';
 import {
   DEFAULT_THEME,
-  getColorV8,
   focusStyles,
-  retrieveComponentStyles
+  retrieveComponentStyles,
+  getColor
 } from '@zendeskgarden/react-theming';
 import { stripUnit } from 'polished';
 
 const COMPONENT_ID = 'tabs.tab';
 
 interface IStyledTabProps {
-  isSelected?: boolean;
-  isVertical?: boolean;
+  $isSelected?: boolean;
+  $isVertical?: boolean;
 }
 
 const colorStyles = ({
   theme,
-  isSelected,
-  isVertical
+  $isSelected,
+  $isVertical
 }: IStyledTabProps & ThemeProps<DefaultTheme>) => {
-  const borderColor = isSelected ? 'currentcolor' : 'transparent';
-  const selectedColor = getColorV8('primaryHue', 600, theme);
+  const borderColor = $isSelected ? 'currentcolor' : 'transparent';
+  const borderBlockEndColor = $isVertical ? undefined : borderColor;
+  const borderInlineColor = $isVertical ? borderColor : undefined;
+
+  const selectedColor = getColor({ theme, variable: 'foreground.primary' });
+  const foregroundColor = $isSelected ? selectedColor : 'inherit';
```

Comment:
@steue @lucijanblagonic The default foreground color was originally set to inherit the parent container's. In most cases (including the Storybook stories) that would be `foreground.default`, which has the benefit of showing a clear distinction between default and disabled tabs.

![Screenshot 2024-06-20 at 4 33 14 PM](https://github.com/zendeskgarden/react-components/assets/6879688/a48ab37f-21b2-44ec-9923-482508874de5)
![Screenshot 2024-06-20 at 4 33 02 PM](https://github.com/zendeskgarden/react-components/assets/6879688/0ea52950-9eb9-4ba2-97c3-81b6a0d44249)

Should we keep it as such or use `foreground.subtle` as shown in Figma instead?

CC: @jzempel

Reply:
Inheriting feels OK here as long as the intent is preserved by the variable (e.g., disabled vs selected vs. default (inherited)). Hopefully I'm understanding this correctly. haha

Reply:
@ze-flo Yes, the `foreground.default` is the preferred color here. We corrected this in design, and we left a note for making the change in the inventory.